src/pudl/output/pudltabl.py

```diff
@@ -802,14 +807,40 @@ def gen_original_eia923(self, update=False):
             )
         return self._dfs["gen_og_eia923"]
 
+    def gen_fuel_allocated_generator_fuel_type_eia923(self, update=False):
+        """Net generation from gen fuel table allocated to generator/prime_mover/fuel."""
+        if update or self._dfs["gen_pm_fuel_eia923"] is None:
+            self._dfs[
+                "gen_pm_fuel_eia923"
+            ] = allocate_generation_fuel_by_generator_fuel_type(pudl_out=self)
+        return self._dfs["gen_pm_fuel_eia923"]
+
     def gen_allocated_eia923(self, update=False):
```

Comment:
Now that there's more than one kind of "allocated" generation this function name is ambiguous.

Reply:
is `gen_fuel_allocated_by_generator_eia923` better?